Assets/Scripts/Spritz/Controller/SpritzController.cs

```diff
@@ -298,6 +294,24 @@ public void Hit(Vector2 hitDirection)
 
         }
 
+        public void OnObjectSpawn()
```

Comment:
this needs to be tested still but i'll cross that bridge when we have respawn logic/requirements for enemies